src/_pytest/deprecated.py

```diff
@@ -4,11 +4,32 @@
 
 Keeping it in a central location makes it easy to track what is deprecated and should
 be removed when the time comes.
+
+All constants defined in this module should be either PytestWarning instances or UnformattedWarning
+in case of warnings which need to format their messages.
 """
 from __future__ import absolute_import, division, print_function
 
+import attr
+
 from _pytest.warning_types import RemovedInPytest4Warning
 
+
+@attr.s
+class UnformattedWarning(object):
```

Comment:
we might want to move this to warning_types - then this module can be only constants

Reply:
Done!